# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cookies/__dir__.ini
# Commit: 3ce54e39ebcd
# Full Hash: 3ce54e39ebcdf069615571d1805af75ad969f006
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-03 04:20:58
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r dff1c1aca50c -r 3ce54e39ebcd testing/web-platform/meta/cookies/__dir__.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/cookies/__dir__.ini	Tue Jun 02 13:50:12 2020 +0000
@@ -0,0 +1,1 @@
+prefs: [network.cookie.sameSite.schemeful:false]